libs/saml2-web-sso/src/SAML2/WebSSO/Types.hs

```diff
@@ -224,6 +224,7 @@ data DeniedReason
   = DeniedStatusFailure
   | DeniedBadUserRefs {deniedDetails :: String}
   | DeniedBadInResponseTos {deniedDetails :: String}
+  | DeniedNoInResponseTo {idpIssuer :: Issuer}
```

Comment:
Is the `Issuer` argument used?

Reply:
no, i thought i'd want to use it, but then realized i didn't, and forgot to remove.  i'll do that separately, thanks!

Reply:
https://github.com/wireapp/wire-server/pull/4513/commits/988fe51d1f2f92444f1ca4f3d4e6b439cb5dc09e